testdata/p4tc_samples_outputs/tc_may_override_example_01_control_blocks.c

```diff
@@ -111,9 +111,18 @@ if (/* hdr->ipv4.isValid() */
                         switch (value->action) {
                             case MAINCONTROLIMPL_IPV4_TBL_1_ACT_MAINCONTROLIMPL_NEXT_HOP: 
                                 {
+                                    if (value->is_default_miss_act) 
+                                    {
+/* send_to_port(p4tc_filter_fields.ipv4_tbl_1_next_hop_vport) */
+                                        compiler_meta__->drop = false;
```

Comment:
this statement and the other common statements in if/else can be moved outside if

Reply:
This required more common code from ebpf to be replicated. Hence implemented like this